### PR TITLE
Session Status Guard

### PR DESCRIPTION
**Pull request by TS-Daniel16**
#326

### CODE REVIEW

File: apps/api/test/user-sessions.e2e-spec.ts [hunk 1]
```diff
@@ -363,17 +363,25 @@ describe('User Sessions (e2e)', () => {
...
 
+      console.log('After Mail');
+
       const { data } = await request(app.getHttpServer())
         .auth(`${UsersData.sessionOwner1.id}`, { type: 'bearer' })
         .mutate(INVITE_USER_SESSION)
         .variables(INVITE_USER_SESSION_VARIABLES)
         .expectNoErrors();
 
+      console.log('After Invite Mutation');
```
Comment (TS-Tino16) [inline apps/api/test/user-sessions.e2e-spec.ts:379]: Console. Log

File: apps/api/test/user-sessions.e2e-spec.ts [hunk 2]
```diff
@@ -363,17 +363,25 @@ describe('User Sessions (e2e)', () => {
...
+
       expect(data).toMatchSnapshot();
```
Comment (TS-Tino16) [inline apps/api/test/user-sessions.e2e-spec.ts:382]: Console.log

**Review by TS-Tino16 — APPROVED** [review]
See comments

File: apps/api/test/user-sessions.e2e-spec.ts (hunk 1 above)
Comment (TS-Daniel16) [reply apps/api/test/user-sessions.e2e-spec.ts]: Removed

File: apps/api/test/user-sessions.e2e-spec.ts (hunk 2 above)
Comment (TS-Daniel16) [reply apps/api/test/user-sessions.e2e-spec.ts]: Removed

**Review by TS-Tino16 — APPROVED** [review]

**Pull request merged**